src/main/java/data/exercises/ExerciseList.java

```diff
@@ -1,27 +1,20 @@
 package data.exercises;
 
-import java.io.File;
-import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Scanner;
 
 public class ExerciseList {
-    private static final String EXERCISE_FILE_PATH = "exercises\\Exercises.txt";
-    private static final File EXERCISE_FILE = new File(EXERCISE_FILE_PATH);
     private final ArrayList<String> exercises = new ArrayList<>();
```

Comment:
Just a suggestion, might want to rename exercises to exercisesList perhaps.

Reply:
Suggestion adopted in Commit ce06262